server/liveblog/blogs/blogs.py

```diff
@@ -51,7 +88,13 @@ def on_create(self, docs):
     def get(self, req, lookup):
         if req is None:
             req = ParsedRequest()
-        return self.backend.get('blogs', req=req, lookup=lookup)
+        docs = super().get(req, lookup)
+        return docs
+
+    def find_one(self, req, **lookup):
+        doc = super().find_one(req, **lookup)
+        get_last_cid(doc)
```

Comment:
cid is a counter which keeps track of my items/posts. It must be kept on blog level so that's why the method get_last_cid is called when one specific blog is requested.
For example: you have a blog, let's say BLOG1 and for this blog you want to create a POST. Your new created post has now cid=1. If you modify the post, the value of the cid will be 2. When you retrieve the BLOG1, among all the details you will also receive cid=2.

Was this description helpful?